helpers/common.go

```diff
@@ -60,9 +60,23 @@ func UnameMachine() (string, error) {
 	return arch, nil
 }
 
+type KernelVersionComparison int
+
+const (
+	KernelVersionOlder   KernelVersionComparison = -1
+	KernelVersionEqual                           = 0
+	KernelVersionNewer                           = 1
+	KernelVersionInvalid                         = -3
```

Comment:
I think we need to repeat the type here since we have assignments, otherwise consts but first will be `const ... untyped int`.

If the const values are not required for outside use, another way can be using iota:

```suggestion
	KernelVersionOlder KernelVersionComparison = iota
	KernelVersionEqual
	KernelVersionNewer
	KernelVersionInvalid
```